src/pages/UserPage/_user_page.scss

```diff
@@ -0,0 +1,110 @@
+@import '../../utils/scss/constants';
+
+.user-page {
+  width: 100vw;
+  height: 100vh;
```

Comment:
tutaj zamiast height dałbym min-height na 100vh, bo jeśli w przyszłości będzie coś dodane do widoku, to przy wysokości ustawionej na sztywno elementy będą na siebie nachodzić, a przy min-height strona się powiększy.

Reply:
Dzięki wielkie, jak wrócę z pracy wdrożę poprawki :)